views/clucks/index.ejs

```diff
@@ -0,0 +1,55 @@
+<%- include('../partials/header') %>
+
+<div class="clucks-list">
+
+  <% for (let cluck of clucks) {
+
+<!-- Calculate time -->
+
+function timeSince(date) {
+
+var seconds = Math.floor((new Date() - date) / 1000);
+
+var interval = Math.floor(seconds / 31536000);
+
+if (interval > 1) {
+return interval + " years";
+}
+interval = Math.floor(seconds / 2592000);
+if (interval > 1) {
+return interval + " months";
+}
+interval = Math.floor(seconds / 86400);
+if (interval > 1) {
+return interval + " days";
+}
+interval = Math.floor(seconds / 3600);
+if (interval > 1) {
+return interval + " hours";
+}
+interval = Math.floor(seconds / 60);
+if (interval > 1) {
+return interval + " minutes";
+}
+return Math.floor(seconds) + " seconds";
+}
+var aDay = 24*60*60*1000
+timeAgo = timeSince(new Date(cluck.createdAt));
+%>
+
+    <div class="container-fluid">
+      <a href="/clucks/<%= cluck.id %>">
+        <img class="img-fluid" src="<%= cluck.image_url %>" />
```

Comment:
What happens if the cluck doesn't have an image provided? Can we hide the tag, or provide a default image instead?